src/components/tree/integration/webview/tree-webview-view-provider.ts

```diff
@@ -0,0 +1,131 @@
+/********************************************************************************
```

Comment:
Thanks for updating the license from EPL-2.0 to MIT here and in other files to be compliant with the repository's license. Approved from my side.